apps/site/pages/docs/guides/contextual-images.mdx

```diff
@@ -0,0 +1,115 @@
+# Contextual Images
```

Comment:
This is a "How-to" guide in the way that @emersonlaurentino proposed we do in the past. I did this for my benefit as well. It's not necessary but I think it's a good idea!